Modules/CitiesViewController/Cells/CityTableCell.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  CityTableCell.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 04/04/2024.
+//
+
+import UIKit
+
+final class CityTableCell: UITableViewCell {
+    
+    lazy var labelsStack = { 
+        let stack = UIStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        stack.spacing = 10
+        return stack
+    }()
+    
+    var pageTitleLbl = ReusableBoldLabel()
+    var pageSecondaryLbl = ReusableBoldLabel() {
+        didSet {
+            pageSecondaryLbl.isHidden = true
+        }
+    }
+
+    static let identifier = "CityTableCell"
```

Comment:
You can use the class name instead of the hardcoded identifier. 

Example: 
- `String(describing: CityTableCell.self)`
- `"\(CityTableCell.self)"`